demosplan/DemosPlanCoreBundle/Validator/PrePersistUniqueInternIdConstraintValidator.php

```diff
@@ -31,11 +31,9 @@ public function validate($value, Constraint $constraint): void
     {
         $value = $this->validateType($value, $constraint);
 
-        $occupiedInSavedEntities = $this->isIdOccupiedInSavedEntities($value);
         $occupiedByLoadedEntities = $this->isIdOccupiedByLoadedEntities($value->getInternId(false), $value);
-        if ($occupiedInSavedEntities || $occupiedByLoadedEntities) {
+        if ($occupiedByLoadedEntities) {
             $this->context->buildViolation($constraint->message)
-                ->setParameter('{{ internId }}', $value->getInternId())
                 ->addViolation();
         }
```

Comment:
Why was this validation logic simplified.

Reply:
What should it look like?

Reply:
I'd prefer we keep the `$occupiedInSavedEntities` check.

Reply:
done in [ac92fca](https://github.com/demos-europe/demosplan-core/pull/2575/commits/ac92f3afaa6cb926ce7e77f44f24df9dc6dc82af)